guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -26,11 +26,17 @@ You must assign every host that is managed by {Project} to a domain. Using the d
 Subnet::
 You must assign every host managed by {Project} to a subnet. Using subnets, {Project} can then manage IPv4 reservations. If there are no reservation integrations, you still must create and associate at least one subnet. When you manage a subnet in {Project}, you cannot create DHCP records for that subnet outside of {Project}. In {Project}, you can use IP Address Management (IPAM) to manage IP addresses with one of the following options:
 +
-* *DHCP*: the DHCP {SmartProxy} manages the assignment of IP addresses by finding the next available IP address starting from the first. Before assigning an IP address, {SmartProxy} sends an ICMP and TCP ping to verify whether the IP address is in use.
-* *Internal DB*: {Project} manages a list of available IP addresses in the PostgreSQL database. {Project} assigns the IP addresses automatically.
-* *Random DB*: {Project} manages a list of available IP addresses in the PostgreSQL database, but randomizes the results to avoid IPv4 race conditions.
+* *DHCP*: DHCP {SmartProxy} manages the assignment of IP addresses by finding the next available IP address starting from the first address of the range skipping all addresses that are reserved. Before assigning an IP address, {SmartProxy} sends an ICMP and TCP pings to check whether the IP address is in use. Note that if a host is powered off, or has a firewall configured to disable connections, {Project} makes a false assumption that the IP address is available. Therefore, use the *DHCP* option with subnets that are in full {Project} control and that do not have hosts created externally. This check does not work for hosts that are turned off, so the DHCP option can only be used for subnets that {Project} controls and that do not have any hosts created externally.
++
+{SmartProxy} DHCP module retains offered IP addresses for a short period of time to prevent collisions during concurrent access, therefore some IP addresses in the IP range might remain temporary unused.
```

Comment:
```suggestion
The {SmartProxy} DHCP module retains the offered IP addresses for a short period of time to prevent collisions during concurrent access, so some IP addresses in the IP range might remain temporarily unused.
```